### ISSUE TITLE
Separate plugin for alternating row colors

### ISSUE THREAD

**Issue by mjthoraval** [MuiseDestiny/zotero-style#785]
Thank you very much for this plugin. It seems to have a lot of useful features.
I have tried several times to use it in the last few months, but everytime I install it on Zotero 7 Beta, Zotero become extremelly slow, completely unsable. I need to restart Zotero in safe mode to disable the plugin before I can do anything.

I have more than 15k items in my library, so I understand that some of the feauters of this plugin are probably not suitable for a large library. So it would be great if I could use some of the features separately from all the other features.

Especially, I am interested in using the alternating row colors on Winodws, similar to MacOS or the web library, as described here:
https://forums.zotero.org/discussion/98500/feature-request-alternating-background-color-for-library-entries-in-windows

Would it be possible to offer that feature as a separate plugin?
Thank you very much.

**Comment by TW96** [comment]
I agree that the alternating row colors in Windows is a really nice feature. But as you say this plugin does seem to slow Zotero down quite a lot. I know that any plugin features not being used can be turned off in Preferences, but I wonder if there are code optimizations that could speed it up ? It would be a pity if all the nice features became too hard to use.
